core/modules/telemetry/telemetry.telemetry.inc

```diff
@@ -23,6 +23,11 @@ function telemetry_telemetry_info() {
     'description' => t('The operating system of your server, such as RedHat Linux, Debian, Windows, etc.'),
     'project' => 'backdrop',
   );
+  $info['web_server'] = array(
+    'label' => t('Web Server'),
```

Comment:
Does this need to be Title Case?

```suggestion
    'label' => t('Web server'),
```

Reply:
Officially yes I think it's supposed to be title case.